_source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Send Kubernetes metrics with Helm and the OpenTelemetry Collector
+logo:
+  logofile: k8s-helm.svg
+  orientation: horizontal
+data-source: Kubernetes over Helm with OpenTelemetry
+open-source:
+  - title: logzio-helm-opentelemetry  #is this correct?
+    github-repo: logzio-otel-k8s-metrics  #is this correct? Should it be https://github.com/logzio/logzio-helm/tree/master/charts/opentelemetry
```

Comment:
@yotamloe : Please review and update these fields.

Reply:
@YajB I have updated the requested fields :)